actions/websocket_actions.jsx

```diff
@@ -427,9 +427,17 @@ function handleChannelConvertedEvent(msg) {
     }
 }
 
-function handleChannelUpdatedEvent(msg) {
-    const channel = JSON.parse(msg.data.channel);
-    dispatch({type: ChannelTypes.RECEIVED_CHANNEL, data: channel});
+export function handleChannelUpdatedEvent(msg) {
```

Comment:
@hmhealey trying to figure how this function is invoked/dispatched, was this change made to be consistent with mm-redux? As I understand it, doesn't look like this action is dispatched.

Reply:
Hmm, that's a very good catch. This should be dispatched, but that's clearly not happening. These changes actually made this worse since no channel update events are handled now. I'll file a new ticket for this since it's a different issue than this was meant to fix.

Ideally, I'd like for all of these actions to be dispatched explicitly since I want to remove as much usage of the global dispatch as possible, but that obviously requires properly dispatching them in the first place. :p

Reply:
agreed.